rfcs/testdriver_set_time_zone.md

```diff
@@ -0,0 +1,21 @@
+# RFC 73 : Add testdriver.js support for the `timeZone` WebDriver command.
+
+## Summary
+Add testdriver.js support for the `timeZone` WebDriver extension, which will
+allow setting the current time zone in the browser. This extension is specced
+as part of the [HTML spec](https://github.com/whatwg/html/pull/3047).
+
+## Details
+Changing the time zone is not available via normal web APIs. To test the effect
+of changing time zone on the `ontimezonechange` event there needs to be a way
+to instruct the browser to act as if the time zone is changing: this WebDriver
+extension provides that ability.
```

Comment:
Worth mentioning that this command is also useful to test PWAs that take into account the user's local time zone? IOW, it's useful even if `timezonechange` didn't exist.

Reply:
Are you thinking from the context of WPT still, or from the context of a web-developer (or PWA developer) who might want to test their site/app?

Reply:
Oh yeah, I was talking about the WebDriver command, not the testdriver.js change. Ignore me! :)

Reply:
Not at all - that info helps inform us that this WebDriver extension is probably useful outside of the scope of WPT. Those tend to be even easier to accept, since there's a valid use for them in the 'real world' :)